tools/cockpit.spec

```diff
@@ -52,6 +52,7 @@ URL:            https://cockpit-project.org/
 Version:        0
 Release:        1%{?dist}
 Source0:        https://github.com/cockpit-project/cockpit/releases/download/%{version}/cockpit-%{version}.tar.xz
+Source1:        cockpit-sysusers.conf
```

Comment:
Having to duplicate and keep this in sync with upstream is really ugly, let's avoid that. The whole point of moving to sysusers is to *drop* downstream packaging stuff IMHO, otherwise we could just leave it as-is.